recipes/eigenpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "eigenpy" %}
+{% set version = "1.5.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/stack-of-tasks/eigenpy
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+  host:
+    - eigen
+    - boost 1.69.*
```

Comment:
Does this pin need need to be this precise

Reply:
There was issues with boost 1.70 that are being resolved (fixed in the `develop` branch of `eigenpy` but not yet released), so the pin was a temporary thing for making sure that I could get a green CI in the mean time. I think the main constraint is that `host` and `run` boost versions are identical.

By the way I read https://conda-forge.org/docs/maintainer/pinning_deps.html and I was able to generate a few different packages for different python and boost versions locally with a `conda_build_config.yaml`. The thing I am not so sure about is how to get conda-forge to build packages for different versions of the dependencies (boost for example but also python).
- does it all happen by magic once we get our own feedstock (in this PR it seems like only one version of python 3.7 is tested for example)?
- Should we tackle this once we have our own feedstock, by issuing subsequent PRs where we vary the boost versions (I have seen this in other projects PRs)? 
- is there another mechanism e.g. by adding a `conda_build_config.yaml` in this repo?